src/ProjectTemplates/test/Infrastructure/TemplateTests.props.in

```diff
@@ -18,7 +18,6 @@
 
     <KnownFrameworkReference
       Update="Microsoft.AspNetCore.App"
-      TargetFramework="${DefaultNetCoreTargetFramework}"
```

Comment:
CC @dougbu @JunTaoLuo  - I'd like to find a better workaround than this once the preview1 builds are done - adding a condition like the following:

`Condition="'%(TargetFramework)' == '${DefaultNetCoreTargetFramework}'"`

Gave the following msbuild error:

> error MSB4191: The reference to custom metadata \"TargetFramework\" at position 1 is not allowed in this condition \"'%(TargetFramework)' == 'netcoreapp3.1'\".

Whereas keeping the TFM update caused an error due to duplicate entires for `Microsoft.AspNetCore.App` in `KnownFrameworkReference`

Reply:
Probably needed to be `%(KnownFrameworkReference.TargetFramework)`. Try that when you have a chance.

Reply:
I had that in an earlier commit with the same result: https://github.com/aspnet/AspNetCore/pull/14670/commits/784f4d542b6f49f61d68e4c828acc23d7583c41e